app/assets/stylesheets/styles/components/guides/ui/_actions_modal.css.scss

```diff
@@ -0,0 +1,40 @@
+.actions-modal {
+  /*position: fixed;*/
+  /*font-family: Arial, Helvetica, sans-serif;*/
+  /*top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  background: rgba(0,0,0,0.5);
+  z-index: 99999;
+  opacity:0;*/
+  -webkit-transition: opacity 200ms ease-in;
+  -moz-transition: opacity 200ms ease-in;
+  transition: opacity 200ms ease-in;
+  pointer-events: none;
+}
+
+.actions-modal:target {
+  opacity:1;
+  pointer-events: auto;
+}
+
+.actions-modal > div {
+  /*width: 600px;*/
+  /*position: relative;*/
+  /*margin: 20% auto;*/
+  padding: 1rem;
+  border-radius: 5px;
+  background: #f1f1f1;
```

Comment:
Color literals like `#f1f1f1` should only be used in variable declarations; they should be referred to via variable everywhere else.